samples/sampler/stories/default/Marquee.js

```diff
@@ -6,55 +8,64 @@ import {storiesOf} from '@storybook/react';
 import Marquee from '@enact/agate/Marquee';
 import {I18nContextDecorator} from '@enact/i18n/I18nDecorator';
 
-const I18nMarqueeBase = ({rtl}) => {
-	// fn to parse the padding value which is invoked later to keep the knob ordered
-	const spacing = () => {
-		const value = text('marqueeSpacing', Marquee, '50%');
-		if (value && value.indexOf('%') > 0) {
-			return value;
-		}
-
-		return Number.parseInt(value);
-	};
+Marquee.displayName = 'Marquee';
+const Config = mergeComponentMetadata('Marquee', Marquee);
 
-	const marqueeText = rtl ? 'قفز الثعلب البني السريع فوق الكلب الكسول. طائر الفاصوليا يطير عند غروب الشمس.' :
-		'The quick brown fox jumped over the lazy dog. The bean bird flies at sundown.';
+const I18nMarqueeBase = kind({
+	name: 'I18nMarquee',
 
-	const disabled = boolean('disabled', Marquee);
+	propTypes: {
+		rtl: PropTypes.bool
+	},
 
-	return (
-		<section>
+	render: ({rtl, ...rest}) => {
+		const marqueeText = rtl ? 'قفز الثعلب البني السريع فوق الكلب الكسول. طائر الفاصوليا يطير عند غروب الشمس.' : 'The quick brown fox jumped over the lazy dog. The bean bird flies at sundown.';
```

Comment:
I don't know what I did wrong, but even though I change the locale to ar-SA, this string doesn't appear.

Reply:
I'm afraid I don't know what is causing this issue...
This bug is reproducing on my machine, but it works fine on my colleague's. Could someone else from your team check if this bug appears?

Reply:
We all can see only English.
Maybe it's a sampler issue or an I18nContextDecorator issue. Could you make a new jiraishu to remember?
 If it reappears after the story-book update, then I think we should see it.